libs/stats/odc/stats/_gm_ls_bitmask.py

```diff
@@ -25,19 +23,32 @@ def __init__(
             self,
             bands: Optional[Tuple[str, ...]] = None,
             mask_band: str = "QA_PIXEL",
+            # provide flags with high cloud bits definition
+            flags: Dict[str, Optional[Any]] = dict(
+                cloud="high_confidence",
+                # cloud_shadow="high_confidence",
+                cirrus="high_confidence",
+            ),
+            nodata_flags: Dict[str, Optional[Any]] = dict(nodata=False),
             filter: Optional[Dict[str, int]] = None, # dict(closing=int,opening=int,dilation=int)
-            aux_names=dict(smad="sdev", emad="edev", bcmad="bcdev", count="count"),
+            aux_names=dict(smad="smad", emad="emad", bcmad="bcmad", count="count"),
             resampling: str = "nearest",
             work_chunks: Tuple[int, int] = (400, 400),
+            scale: float = 1/0.0000275,
+            offset: float = -0.2/0.0000275,
```

Comment:
default to `scale = 0.0000275` and `offset = -0.2 ` should be good now that we don't try to scale output to a different 10000 scaling.